README.md

```diff
@@ -73,8 +73,8 @@ See demos of running local LLMs *on Intel Core Ultra iGPU, Intel Core Ultra NPU,
       </a>
     </td>
     <td>
-      <a href="https://llm-assets.readthedocs.io/en/latest/_images/arc_llama3-8B_fp8_textwebui.gif" target="_blank">
-        <img src="https://llm-assets.readthedocs.io/en/latest/_images/arc_llama3-8B_fp8_textwebui.gif" width=100%; />
+      <a href="https://llm-assets.readthedocs.io/en/latest/_images/arc_qwen2.5-14B_q4_0_llamacpp.gif" target="_blank">
```

Comment:
https://llm-assets.readthedocs.io/en/latest/_images/arc_qwen2.5-14B_q4_0_llamacpp.gif 
404, can not be found

Reply:
should be qwen2_5?